### ISSUE TITLE
Add additional properties for pagination

### ISSUE THREAD

**Issue by vibhav2799** [Stutive/Stutive#5]
As title, next, prev properties.

@AlpriElse

**Comment by AlpriElse** [comment]
If you've already started on this, go ahead with dev. But, I'd put this as lower priority than filtering

**Comment by vibhav2799** [comment]
Yeah, low priority. I am creating tickets for stuff outside of current vision.
